spec/models/document_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Document, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of(:name) }
+  end
+
+  describe 'associations' do
+    document = FactoryBot.create(:document)
```

Comment:
i think its better to use rspec let here rather than using variable

Reply:
I refactored it, please check